<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BionicBenchClass</class>
 <widget class="QMainWindow" name="BionicBenchClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BionicBench</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1321</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabProfile">
     <attribute name="title">
      <string>PC Profile</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabTests">
     <attribute name="title">
      <string>Stress Tests</string>
     </attribute>
     <widget class="QProgressBar" name="dataProgressBar">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>440</y>
        <width>341</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="floatProgressBar">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>440</y>
        <width>341</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>80</y>
        <width>301</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Transfer Stress Test</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="floatTitleLabel">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Floating Point Stress Test</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="intStartButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QPushButton" name="floatStartButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QLabel" name="intTitleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Integer Operation Stress Test</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>540</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Run All Tests</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="intProgressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>341</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>301</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="FloatingPointOperationTest">
       <item>
        <widget class="QScrollArea" name="floatScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>297</width>
            <height>337</height>
           </rect>
          </property>
          <widget class="QTextEdit" name="floatTextEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>301</width>
             <height>341</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="dataStartButton">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>100</y>
        <width>301</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="DataTransferOperationTest">
       <item>
        <widget class="QScrollArea" name="dataScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>297</width>
            <height>337</height>
           </rect>
          </property>
          <widget class="QTextEdit" name="dataTextEdit">
           <property name="geometry">
            <rect>
             <x>-7</x>
             <y>0</y>
             <width>311</width>
             <height>341</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>301</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="IntegerOperationTest">
       <item>
        <widget class="QScrollArea" name="intScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>297</width>
            <height>337</height>
           </rect>
          </property>
          <widget class="QTextEdit" name="intTextEdit">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>301</width>
             <height>341</height>
            </rect>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1329</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BionicBench.qrc"/>
 </resources>
 <connections/>
</ui>
